src/types/zod-fixture.d.ts

```diff
@@ -0,0 +1,41 @@
+declare namespace ZodFixture {
```

Comment:
Why do we need to define our own type for this?

Reply:
We don't, but I found it easier to use with documentation.

```
#region defaults
interface Defaults {
   // ....
}
#endregion defaults
```

Looks better than:

```ts
export default {
  // .....
}
```ts

If we use it as a snippet.

I'm not attached to this and am happy to explore other options.

Reply:
What about just defining the interface in the defaults file?
Another note: is there a reason why `prettify` isn't a boolean (that isn't optional)?

Reply:
The reason for this is that I prefer to keep "definition types" for external libraries.
For the types we're in control of, I prefer to define an interface or type for it.

Reply:
I agree that we should be consistent, I just couldn't find a good way to do this:

```ts
#region defaults
interface Defaults {
  // .....
}
#endregion defaults

export Defaults // ERROR: typescript hates this
```

We'd end up having to do
```ts
#region defaults
export interface Defaults {
  // .....
}
#endregion defaults
```